### PR TITLE
Remove Plan Management for Partner Products

### PR DESCRIPTION
**Pull request by cbauerman**
#### Changes proposed in this Pull Request

* Remove References to renewal time, being able to cancel or renew, or plan cost from Calypso for Partner Jetpack Plans

##### Deliberately Omitted

* Links directly to partners/hosts in place of various actions removed

##### Screenshots

`/me/purchases`

**BEFORE**
purchase list before
**AFTER**
purchase list after

`/me/purchases/:siteSlug/:purchaseId`

**BEFORE**
specific purchase before

**AFTER**
specific purchase after

`/plans/:siteSlug`

**BEFORE**
plans screen before

**AFTER**
plans screen after

`/plans/my-plan/:siteSlug`

**BEFORE**
my plan before

**AFTER**
my plan after

#### Testing instructions

1. Aquire a site with a partner Jetpack plan and a site with a regular Jetpack plan
2. For each of the following sites
   * Your Partner Plan Site on Production - these should match the "Before" Screenshots
   * Your Partner Plan Site on the branch - these should match the "After" Screenshots
   * Your "regular" Plan Site on the branch - these should match the "Before" Screenshots
3. Visit each of this pages
   * `/me/purchases`
   * `/me/purchases/:siteSlug/:purchaseId`
   * `/plans/:siteSlug`
   * `/plans/my-plan/:siteSlug`

### CODE REVIEW

File: client/me/purchases/manage-purchase/style.scss [hunk 1]
```diff
@@ -201,7 +206,7 @@
 			}
 			&:first-child:nth-last-child( 4 ),
 			&:first-child:nth-last-child( 4 ) ~ li {
-				  max-width: 25%;
+				max-width: 25%;
```
Comment (cbauerman) [inline client/me/purchases/manage-purchase/style.scss]: prettier did this

**Comment by folletto** [comment]
The visual for `/plans/:siteSlug` seems different from the designs, it's spaced oddly and very stretched. I checked the code and it seems using `EmptyContent` as expected, so I think it's the image used: the SVG seems wide (it's named `jetpack-header.svg` so I think it's meant to be used elsewhere). Try replacing with `illustration-jetpack.svg`

It's also missing the button to go to the host. Maybe the data to show that isn't there? It would be ideal to always provide a "next action" for the user.

Any chance we can fix that? 

Screenshot 2019-10-16 at 10 52 30

**Comment by cbauerman** [comment]
> The visual for `/plans/:siteSlug` seems different from the designs, it's spaced oddly and very stretched. I checked the code and it seems using `EmptyContent` as expected, so I think it's the image used: the SVG seems wide (it's named `jetpack-header.svg` so I think it's meant to be used elsewhere). Try replacing with `illustration-jetpack.svg`
> 
> It's also missing the button to go to the host. Maybe the data to show that isn't there? It would be ideal to always provide a "next action" for the user.
> 
> Any chance we can fix that?
> 

The button is technically out of reach at the moment, because we don't have the ability to figure out where to send them.

As for the illustrations I totally missed the correct illustration when looking for it 🤦‍♂️ fixed in #36875

**Comment by folletto** [comment]
> The button is technically out of reach at the moment, because we don't have the ability to figure out where to send them.

Noted! :)

**Pull request merged**